R/epinow.R

```diff
@@ -458,18 +459,21 @@ run_region <- function(target_region,
                        return_estimates,
                        return_partial_estimates,
                        ...) {
-  futile.logger::flog.info("Initialising estimates for: %s", target_region)
+  futile.logger::flog.info("Initialising estimates for: %s", target_region, 
+                           name = "EpiNow2.epinow")
```

Comment:
This string is being used a lot - perhaps tuck it onto a variable?

Reply:
the logger name?

Reply:
yes, "EpiNow2.epinow"

Reply:
can do - think save that for an extension to the logging system in a future PR.